can_tools/scrapers/official/DE/de_vaccine.py

```diff
@@ -144,6 +144,7 @@ def normalize(self, data: Dict[str, requests.models.Response]) -> pd.DataFrame:
                 vintage=self._retrieve_vintage(),
                 location_name=lambda row: row["location_name"].str.replace("-", " "),
                 age=lambda row: row["age"].str.replace("+", "_plus"),
+                dt=lambda row: row["dt"].dt.date
```

Comment:
The date column was throwing an error about datatypes (specifically `datetime64[ns, UTC]`), so this forces it into a workable format